### PR TITLE
Docker-DB

### PR DESCRIPTION
**Pull request by igMike-V**
- added a docker-compose.yml to create a development database.  
- added readme to explain setup to devs new to the project
- added routes to get all community-events and single events. (this may already be done in an open PR so if thats the case I can revert)
changed the variable name of err to e in controller file to match the other methods.

### CODE REVIEW

File: server/database-dev/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: '3.1'
+
+services:
+
+  db:
```
Comment (0916dhkim) [inline server/database-dev/docker-compose.yml:5]: #### Suggestion (non-blocking)
We can enable verbose logging by adding this:
```
command: ["postgres", "-c", "log_statement=all"]
```
https://stackoverflow.com/a/58806511/7566087

File: server/src/modules/community-event/controller.ts [hunk 2]
```diff
@@ -34,8 +34,25 @@ const communityEventController = {
 
       // serialize response
       res.json({ data: communityEventSerializer.default(response) });
-    } catch (err) {
-      next(err);
+    } catch (e) {
```
Comment (0916dhkim) [inline server/src/modules/community-event/controller.ts:37]: Is this related to this PR?

File: server/database-dev/README-dev-db.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Setting up postgres Database on for development.
+If you do not have docker installed on your system do it now:
+Docker: [https://www.docker.com/]
+Docker Docs: [https://docs.docker.com/]
+
+1. add the following line to your .env file "/server/.env"  
+```DATABASE_URL=postgres://eotuser:eotdevpassword@localhost:5432/eotdb```
+2. Navigate to ```/server/database-dev``` directory
```
Comment (0916dhkim) [inline server/database-dev/README-dev-db.md:8]: #### Suggestion (non-blocking)
I think it is okay to put `docker-compose.yml` file directly under `/server`. Either way is fine to me.

**Review by 0916dhkim — APPROVED** [review]
LGTM!
Couple minor suggestions but they are not blocking. You can go ahead and merge this PR when you are happy with it.

File: server/src/modules/community-event/controller.ts (hunk 2 above)
Comment (igMike-V) [reply server/src/modules/community-event/controller.ts:37]: AH no... I changed it because the rest of the controllers used e instead of err.  I hadn't planned to touch any backend files but when I went to test the connection to the front end I noticed that there was no 'get' handler on the community events route.

File: server/database-dev/README-dev-db.md (hunk 3 above)
Comment (igMike-V) [reply server/database-dev/README-dev-db.md:8]: Will there be a production docker built into the CI?   if so might be best to rename this to docker-compose-dev.yml now.

Comment (0916dhkim) [reply server/database-dev/README-dev-db.md:8]: Production server & db will not be dockerized, at least for now!
I'm down for renaming the file to `docker-compose-dev.yml`.

File: server/src/modules/community-event/controller.ts (hunk 2 above)
Comment (0916dhkim) [reply server/src/modules/community-event/controller.ts:37]: Got it. Those changes look good too, so it is okay to merge with the docker changes :+1:

**Review by anjolaoluwaakindipe — APPROVED** [review]

**Comment by igMike-V** [comment]
Merging with a couple of changes to standardize some naming conventions controllers and added cors dependency to the backend for frontend integration.
